README.md

```diff
@@ -139,7 +139,9 @@ Available placeholder types:
 * `DateTime`
 
   Placeholders with type `DateTime` must have a format specified. The valid values are the names of
-  [the `DateFormat` constructors][dateformat-constructors], e.g. `yMd`, `jms`, or `EEEEE`.
+  [the `DateFormat` constructors][dateformat-constructors], e.g., `yMd`, `jms`, or `EEEEE`.
+  As of Flutter 3.29.0, the `+` character can be utilized to merge date and time formatting information, for instance, `yMd+jms`.
+  This allows specifying both date and time formats in a single placeholder.
```

Comment:
```suggestion
  This allows - for example - specifying both date and time formats in a single placeholder.
```

The possibilities are much more, like adding weekday, quarter, standalone month etc.